web_copy_confirm/static/src/js/web_copy_confirm.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2018 DynApps <http://www.dynapps.be>
+// @author Stefan Rijnhart <stefan@opener.amsterdam>
+// @author Robin Conjour <rconjour@demolium.com>
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+odoo.define("web_copy_confirm.web_copy_confirm", function (require) {
+    "use strict";
+
+    const Core = require("web.core");
+    const FormController = require("web.FormController");
+    const Dialog = require("web.Dialog");
+    const _t = Core._t;
+
+    return FormController.include({
```

Comment:
The `return` is also not necessary here I believe?

Reply:
Indeed, you were right.

Reply:
Thanks for putting it back!